Client/YooChooseNotifier.php

```diff
@@ -140,19 +141,17 @@ public function updateContent($contentId)
             return;
         }
 
-        if (isset($this->logger)) {
-            $this->logger->info("Notifying YooChoose: updateContent($contentId)");
+        $this->log(sprintf('Notifying Recommendation Service: updateContent(%s)', $content->id));
+
+        $notification = array();
```

Comment:
Lines from 146-149 repeat three times, so maybe we can extract this to the separate method:  

```php
private function generateNotifications($action, $content, $languages)
{
        $notification = array();
        foreach ($this->getLanguageCodes($content) as $lang) {
            $notification[] = $this->getNotificationContent(self::ACTION_DELETE, $content, $lang);
        }
       return $notification;
}
```